src/test/java/net/snowflake/ingest/TestUtils.java

```diff
@@ -24,17 +25,13 @@
 import java.security.KeyPairGenerator;
 import java.security.PrivateKey;
 import java.security.spec.PKCS8EncodedKeySpec;
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.ResultSet;
-import java.sql.Statement;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Optional;
-import java.util.Properties;
+import java.sql.*;
+import java.util.*;
```

Comment:
We do not use * imports (IDE setting). I already got a a lot of comments to revert * imports :)

Reply:
argh, thanks! I changed my IDE settings related to that and reverted the * imports.